zkpylons/controllers/admin.py

```diff
@@ -1185,9 +1185,9 @@ def previous_years_stats(self):
                         years[year] += 1
                     else:
                         years[year] = 1
-                if len(registration.prevlca) == len(lca_rego['past_confs']):
+                if len(registration.prevlca) == len(Config.get('rego', 'past_confs')):
```

Comment:
This is expecting a list, but Config.get just returns string. We'll need to eval it somewhere (which isn't pretty)